### ISSUE TITLE
[Chromaitcraft] Crash when using linker on world rift 

### ISSUE THREAD

**Issue by tiny-maple** [ReikaKalseki/Reika_Mods_Issues#2555]
Crash log: https://pastebin.com/SxVvt16y
Every time I use a linker on a worldrift just placed in the world, I crash. It works fine afterwards.

**Comment by ReikaKalseki** [comment]
That looks like you used it on a CVT/high ratio gear/coil/worm gear.

**Comment by tiny-maple** [comment]
~I used it on a CVT that was coupled to a gas turbine.~

**Comment by ReikaKalseki** [comment]
You said you used it on a world rift.

**Comment by tiny-maple** [comment]
>You said you used it on a world rift.

sorry, i misunderstood that as using world rift on a CVT.

![2020-06-07_16 06 25](https://user-images.githubusercontent.com/62176592/83963462-e95fe680-a8d8-11ea-930b-7e490eac980d.png)

This is my setup. I coupled a cvt to a gas turbine, then i switch the gas turbine on and finally I placed the world rift to the cvt. I crash immediately when I shift+right click the newly crafted world rift with the linker. After a relogin, using the world rift with the linker is fine. This occur with every newly crafted world rift.
